jobs/nfsbroker/spec

```diff
@@ -47,3 +47,19 @@ properties:
   nfsbroker.db_ca_cert:
     default: ""
     description: "(optional) CA Cert to verify SSL connection, if not include, connection will be plain"
+  nfsbroker.allowed-in-source:
+    description: "Comma separated list of white-listed options that may be configured in supported in the mount_config.source URL query params. If a non-white-listed option is received from a user as arbitrary params, then the broker will reject the user request."
+    example: "uid,gid"
+    default: "uid,gid"
+  nfsbroker.default-in-source:
+    description: "Comma separated list of default values for options in the mount_config.source URL query params, formatted as 'option:default'. If an option is not specified in the volume mount, or the option is not white-listed, then the specified default value will be applied."
+    example: "uid:1000,gid:1000"
+    default: ""
+  nfsbroker.allowed-in-mount:
+    description: "Comma separated list of white-listed options that may be configured in supported in the mount_config options. If a non-white-listed option is received from a user as arbitrary params, then the broker will reject the user request."
+    example: "allow_root,allow_other,nfs_uid,nfs_gid,auto_cache,fsname"
+    default: "auto_cache"
+  nfsbroker.default-in-mount:
+    description: "Comma separated list default values for options that may be configured in the mount_config options, formatted as 'option:default'. If an option is not specified in the volume mount, or the option is not white-listed, then the specified default value will be applied."
```

Comment:
please consider replacing "in the volume mount" with "in binding arbitrary params"